content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst

```diff
@@ -108,16 +108,23 @@ invoices and bills by email or post.
 Configuration
 -------------
 
-First, :ref:`install <general/install>` the :guilabel:`Peppol` module (`account_peppol`).
+:ref:`Install <general/install>` the :guilabel:`Peppol` module (`account_peppol`).
 
 .. image:: electronic_invoicing/peppol-module.png
    :alt: Peppol module install
 
-Then, go to :menuselection:`Accounting --> Configuration --> Settings`, tick the
-:guilabel:`Use PEPPOL Invoicing`, and fill in the following information:
+Registration
+------------
```

Comment:
I agree with Chiara, mostly because the installation of a module hardly requires a full section ^^
I'll change slightly the structure before the merge

Reply:
@jcs-odoo probably, but I personally think "registration" is a more accurate word for this part of the documentation :slightly_smiling_face: